www/front_src/src/Login/api/decoder.ts

```diff
@@ -25,3 +25,24 @@ export const platformVersionsDecoder = JsonDecoder.object<PlatformVersions>(
   },
   'Platform versions Decoder',
 );
+
+const providerConfigurationDecoder = JsonDecoder.object<ProviderConfiguration>(
+  {
+    authenticationUri: JsonDecoder.string,
+    id: JsonDecoder.number,
+    isActive: JsonDecoder.boolean,
+    isForced: JsonDecoder.optional(JsonDecoder.boolean),
```

Comment:
What is this syntax ?

Reply:
As the `isForced` type is `boolean | undefined`, `optional` means `undefined`